File: src/targets/clojure/clj_http/fixtures/cookies.clj
(require '[clj-http.client :as client])

<<<<<<< HEAD
(client/get "https://httpbin.org/cookies" {:headers {:cookie "foo=bar; bar=baz"}})
=======
(client/post "http://mockbin.com/har" {:headers {:cookie "foo=bar; bar=baz"}})
>>>>>>> upstream/master

File: src/targets/php/guzzle/fixtures/cookies.php
<?php
<<<<<<< HEAD
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('GET', 'https://httpbin.org/cookies', [
=======

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'http://mockbin.com/har', [
>>>>>>> upstream/master
  'headers' => [
    'cookie' => 'foo=bar; bar=baz',
  ],
]);

echo $response->getBody();